mikeio/spatial/FM_geometry.py

```diff
@@ -476,6 +523,13 @@ def type_name(self):
         """Type name, e.g. Mesh, Dfsu2D"""
         return self._type.name if self._type else "Mesh"
 
+    @property
+    def ndim(self) -> int:
```

Comment:
@jsmariegaard Is there more to it than this?

Reply:
Are you referring to the PR in general or something specific?

Reply:
Only referring to the `ndim` property